it/controllers/partners/PartnerSummaryControllerISpec.scala

```diff
@@ -0,0 +1,144 @@
+
+package controllers.partners
+
+import controllers.partners.PartnerIndexValidation.minPartnerIndex
+import featureswitch.core.config.TaskList
+import itutil.ControllerISpec
+import models.Entity
+import models.api._
+import play.api.http.HeaderNames
+import play.api.libs.json.Json
+import play.api.test.Helpers._
+
+class PartnerSummaryControllerISpec extends ControllerISpec {
+
+  def pageUrl(): String = routes.PartnerSummaryController.show.url
+
+  "GET" when {
+    "the journey has no partner entity types" must {
+      "redirect to the task list controller page" in new Setup {
+        given
+          .user.isAuthorised()
+          .registrationApi.getListSection[Entity](Some(Nil))
+          .audit.writesAudit()
```

Comment:
.audit.writesAuditMerged() and .audit.writesAudit() aren't needed as they're part of isAuthorised() for simplicity afaik